server/src/main/java/com/unosquare/carmigo/constant/AppConstants.java

```diff
@@ -0,0 +1,12 @@
+package com.unosquare.carmigo.constant;
+
+public final class AppConstants
+{
+    private AppConstants() {}
+
+    public static final String SECRET_KEY = "Q2FyLW1pZ28gYXBwbGljYXRpb24gaXMgYXdlc29tZSEgSXQgd2lsbCByZXZvbHV0aW9uaXp" +
+            "lIHRoZSB3YXkgeW91IGdldCBhcm91bmQuIEl0IGlzIGVjby1mcmllbmRseSBhbmQgaGVscHMgeW91ciBwb2NrZXQuIFRoZSBhcHBsaW" +
+            "NhdGlvbiBpcyBxdWl0ZSBzdHJhaWdodHdheSwgeW91IGNhbiBlaXRoZXIgYWR2ZXJ0aXNlIHJpZGVzIG9yIHF1ZXJ5IHRoZW0uIFdoZ" +
+            "W4gaW5zaWRlIHRoZSBjYXIgc2hhcmUsIHlvdSBjYW4gZW5qb3kgdGhlIHJpZGUgYW5kIG1ha2UgbmV3IGZyaWVuZHMuIENvbWUgb25i" +
+            "b2FyZCBhbmQgZXhwZXJpZW5jZSB0aGlzIG5ldyBsaWZlc3R5bGUu";
+}
```

Comment:
It might be worth putting this in properties file.  Typically, these will be replaced in environments when deployed.  If hardcoded into code, then hard to modify/maintain.